pyproject.toml

```diff
@@ -31,6 +31,7 @@ pinecone-client = { version = "^3.2.2", optional = true }
 logfire = { version = ">=0.26.0,<1.0.0", optional = true }
```

Comment:
please include `astrapy` with the correct version and `optional = true` like we do for other optional dependencies.